.vib/aspnet-core/vib-publish.json

```diff
@@ -22,7 +22,7 @@
           "url": "{SHA_ARCHIVE}",
           "path": "/bitnami/aspnet-core"
         },
-        "runtime_parameters": "InNlcnZpY2UiOgogICJwb3J0IjogODAKICAidHlwZSI6ICJMb2FkQmFsYW5jZXIi",
+        "runtime_parameters": "YXBwRnJvbUV4dGVybmFsUmVwbzoKICBlbmFibGVkOiB0cnVlCiAgY2xvbmU6CiAgICByZXBvc2l0b3J5OiAiaHR0cHM6Ly9naXRodWIuY29tL2RvdG5ldC9Bc3BOZXRDb3JlLkRvY3MuZ2l0IgogICAgcmV2aXNpb246ICJtYWluIgogIHB1Ymxpc2g6CiAgICBzdWJGb2xkZXI6ICJhc3BuZXRjb3JlL2Z1bmRhbWVudGFscy9zZXJ2ZXJzL2tlc3RyZWwvc2FtcGxlcy82LngvS2VzdHJlbFNhbXBsZSIKICBzdGFydENvbW1hbmQ6IFsiZG90bmV0IiwgIktlc3RyZWxTYW1wbGUuZGxsIl0Kc2VydmljZToKICB0eXBlOiBMb2FkQmFsYW5jZXIKICBwb3J0czoKICAgIGh0dHA6IDgwCg==",
```

Comment:
Given that we are checking for the automounted folder containing the serviceAccount, I think we should start adding those values to the `runtime_parameters`

Reply:
Added, thanks!